src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (newRobot[value]) {
+      return null;
+    }
+
+    newRobot[value] = key;
+  }
```

Comment:
using `for in` loop will be more performant
so, please replace `for of` to `for in`

Reply:
hello! unfortunately it doesn't work with for in loop